main_app/templates/main_app/login.html

```diff
@@ -58,22 +130,18 @@ <h1 class="display-4" style="text-align: center; color: black; font-family: 'Ban
                         Registered
```

Comment:
- [ ] Change `WELCOME` to `WELCOME TO RESCUE` and apply CSS to it similar to `register` page.
- [ ] Change the image of `login` page to the image present on the `register` page.